style.css

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------- Board Styling ----------------------------------*/
+#board {
+    display: flex;
+    width: 150px;
+    margin: 50px;
+}
+#playerNames{
+    margin: 0
+}
+.cell {
+    display: flex;
+    border: 5px solid black;
+    height: 150px;
+    width: 150px;
+}
+.cell123{
+  display:flex; 
+  flex-direction: column; 
+}
+
+.cell456{
+    display: flex;
+    flex-direction: column;
+}
+
+.cell789 {
+    display: flex;
+    flex-direction: column;
+}
+
+
+
```

Comment:
I know it can be tough with only 29 lines of code that are pretty clear in what they are doing, but we would like to see more comments and notes, even in the CSS and HTML files